src/renderer/components/workspaces/floating-panels/FloatingPanelLayer.tsx

```
@@ -186,73 +187,75 @@ function FloatingPanelWindow({
+                  ) : (
+                    <button
+                      className={styles.panelSelectorButton}
+                      onClick={(event) => {
+                        event.stopPropagation();
+                        setSelectorOpen(true);
+                        requestAnimationFrame(() => selectRef.current?.focus());
+                      }}
+                      aria-label="Open editor selector"
+                      title="Switch editor"
+                    >
+                      ▾
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)